src/planscape/rscripts/forsys.R

```diff
@@ -331,6 +331,67 @@ get_weights <- function(priorities, configuration) {
   return(configuration$weights)
 }
 
+get_number_of_projects <- function(scenario) {
+  # this is hardcoded for now. in the future it
+  # might come from the configuration JSON
+  return(5)
+}
+
+get_min_project_area <- function(scenario) {
+  configuration <- get_configuration(scenario)
+  stand_size <- configuration$stand_size
+  min_area <- 10
+  if (stand_size == "LARGE") {
+    min_area <- 500
+  }
+
+  if (stand_size == "MEDIUM") {
+    min_area <- 100
+  }
+
+  log_info(paste0(stand_size, " chosen. Minimum project area is ", min_area))
+  return(min_area)
+}
+
+get_max_treatment_area <- function(scenario) {
+  configuration <- get_configuration(scenario)
+  budget <- configuration$max_budget
+
+  if (!is.null(budget)) {
+    cost_per_acre <- get_cost_per_acre(scenario)
+    max_acres <- budget / cost_per_acre
+    log_info(
+      paste0(
+        "Budget is configured for ",
+        budget,
+        ". Total acres: ",
+        max_acres
+      )
+    )
+    return(max_acres)
+  }
+
+  if (is.null(budget)) {
+    log_info(
+      paste0(
+        "Budget is null, using max acres to be treated. Total area: ",
+        configuration$max_treatment_area_ratio
+      )
+    )
+    return(configuration$max_treatment_area_ratio)
+  }
+
+  max_acres <- get_min_project_area(configuration$stand_size) * 5
```

Comment:
when do we actually get to this line? you have if statements for budged being null and not null